core/collector.class.inc.php

```diff
@@ -69,6 +74,15 @@ public function __construct() {
 		$this->aSkippedAttributes = [];
 	}
 
+	public static function SetCallItopService($oCurrentCallItopService=null){
```

Comment:
I can't add suggestion but please:
- Add `@since`
- Type the $oCurrentCallItopService parameter
- (Surround "=" with spaces)